packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/create/template.html

```diff
@@ -0,0 +1,24 @@
+<div class="octavia-load-balancer-l7-policy-creation">
+    <h2 data-translate="octavia_load_balancer_create_l7_policy_title"></h2>
+
+    <p
+        class="mb-0"
+        data-translate="octavia_load_balancer_create_l7_policy_description"
+    ></p>
+    <div data-ovh-alert="octavia.alerts.l7Policy.create"></div>
+    <div class="">
```

Comment:
```suggestion
    <div>
```
It might be unnecessary to wrap your spinner and your component into a parent `div`